Base level creation

burtybob opened the pull request.

burtybob committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into base-level-creation

burtybob committed:
Base level

Started putting the scene together

kinbiko committed:
Added rigid body to character, and colliders. Also made the background devRant-purple

burtybob committed:
Further level building

Extended the level platforms to be near enough complete. Also added bug
enemy sprite.

burtybob committed:
Made coin background transparent

Made coin background transparent and cleaned up the platforms

burtybob committed:
Tweaked coin color, added flag

Tweaked the coin color to match the face also added the ++ and added the
flag at the end of the level

kinbiko commented:
The ++coins have an extra, almost invisible + near the first plus. I assume that wasn't intentional?
Also the name of this file may not be 100% halal. Suggest changing `+` to `plus`

burtybob committed:
Coin fixes

Fixed a couple of coin issues. One where there was an invisible + and
another where there were some thin parts that caused a disjointed look

burtybob commented:
No it wasn't intentional, probably from where I put the + in the wrong place and cut it. I've fixed that and another issue I noticed, justed pushed them
